test/test.js

```diff
@@ -239,6 +239,38 @@ function clientServerTests(dbName) {
       });
     });
 
+    it('only transition replication when db has adapter', function () {
+      var adapter = (process.browser ? 'websql' : 'leveldb');
```

Comment:
presumably this should fail our tests on Firefox, since Firefox has websql. I'm kinda surprised it's passing actually